painttiming.bs

```diff
@@ -160,6 +164,13 @@ An [=element=] |el| is <dfn>paintable</dfn> when all of the following apply:
 
 <dfn export>First contentful paint</dfn> entry contains a {{DOMHighResTimeStamp}} reporting the time when the user agent first rendered a [=document=] which includes at least one [=element=] that is both [=contentful=] and [=paintable=].
 
+A [=browsing context=] |ctx| is <dfn>paint-timing eligible</dfn> when one of the following apply:
+* |ctx| is a [=top-level browsing context=].
+* |ctx| is a [=nested browsing context=], and the user agent has configured |ctx| to report paint timing.
+
+    NOTE: this allows user-agents to enable paint-timing only for some of the frames, in addition to the main frame, if they so choose.
+    For example, a user-agent may decide to disable paint-timing for cross-origin iframes.
```

Comment:
Might be good to clarify why we're singling out nested-browsing contexts (so add an "in cases their paint times can reveal information about other frames" or somesuch)